tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -18,8 +18,19 @@ Suite Setup         RHOSi Setup
 Resource            ../../../Resources/Page/HybridCloudConsole/HCCLogin.robot
 Resource            ../../../Resources/Common.robot
 
-
 *** Test Cases ***
+Verify Dashbord has no message with NO Component Found
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1493
+    [Documentation]   Verify the service appears in Applications > Enabled
+    [Setup]   Test Setup For Rhods Dashboard
+    Oc Apply  kind=Subscription  src=tests/Tests/100__deploy/100__installation/bad_subscription.yaml
+    Menu.Navigate To Page    Applications    Enabled
+    Sleep    2s
```

Comment:
was the issue happening immediately after the bad subscription was created?
I'd test this code in the RHODS version where the issue was present to check if the code would capture the issue

Reply:
it needs sleep time 10s to load the page hence I have increased the time 
also, I have verified with the old  version where the issue was present, 
looks like the code is working fine 
Links https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1109/console

Reply:
nice, thanks!